File: (A) Assignment/A8-111502036/A8-111502036/A8-111502036.py
"""
Assignment 8 python
Name: 沈哲寬
Student Number: 111502036
Course: 2022-CE1001-B
"""
def preprocess():
    with open('input.txt', 'r') as file:
        data = list(map(int, file.read().split()))
    return data
def showlist(count, listname):
    print('第', count, '次', listname)
def show_original(listname):
    print('原資料：', listname)
def bubble_sort(List):
    n = len(List)
    for j in range(n-1):
        for i in range(n-1):
            if(List[i]>List[i+1]):
                temp = List[i]
                List[i] = List[i+1]
                List[i+1] = temp
        showlist(j+1,List)
    return List
List = preprocess()
Origin = List[:]
bubble_sort(List)
show_original(Origin)

File: (A) Assignment/CA8-111502036/A8-111502036/A8-111502036.cpp
/*
Assignment 8 C++
Name: 沈哲寬
Student Number: 111502036
Course: 2022-CE1001-B
*/
#include<iostream>
using namespace std;
int gcd(int a, int b){
    if(b==0){
        return a;
    }
    return gcd(b,a%b);
}
int lcm(int a,int b){
    int ans = (a*b)/gcd(a,b);
    return ans;
}
int main(){
    int number1,number2;
    cin >> number1 >> number2;
    int ans = lcm(number1,number2);
    cout << ans <<endl; 
}
